crates/sui-faucet/src/faucet/simple_faucet.rs

```diff
@@ -223,6 +224,25 @@ impl SimpleFaucet {
         }))
     }
 
+    /// Clear the WAL list in the faucet
+    pub async fn clear_wal_coins(&self) -> Result<(), FaucetError> {
```

Comment:
I would maybe name this `retry_wal_coins` because it's not guaranteed that they will clear.